Automattic/newspack-app-shell
# Link to non-AMP on persistent element click

_Note: this has to wait until AMP plugin v.1.5.0 is officially released (with `rel="noamphtml"` support)._

## What it does

If rendered on an AMP page, the persistent element links to the non-AMP version. 
Closes #6 

## How to test 

1. with the persistent element set, visit an AMP version of any page
1. click on the persistent element - it should link to the non-AMP version of the page
1. navigate around the site - observe that you're still in the non-AMP version of the site
